hw/top_earlgrey/data/ip/chip_entropy_src_testplan.hjson

```diff
@@ -179,5 +179,74 @@
       tests: []
       bazel: []
     },
+    {
+      name: chip_sw_entropy_src_fw_observe_many_contiguous
+      desc: '''Verify that the software can observe samples fast enough to extract many contiguous samples.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Configure realistic health test threshold values for FIPS/CC compliant mode.
+            - Enable ENTROPY_SRC in FIPS mode (FIXME: is it necessary to use FIPS mode?).
```

Comment:
I would enable it in FIPS mode since the boot-time mode doesn't supply enough entropy.